build-tools/pnpm-lock.yaml

```diff
@@ -64,7 +67,7 @@ importers:
       '@fluidframework/bundle-size-tools': workspace:*
       '@fluidframework/eslint-config-fluid': ^2.0.0
       '@microsoft/api-extractor': ^7.35.3
-      '@oclif/core': ^2.8.6
+      '@oclif/core': ~2.4.0
```

Comment:
Why the oclif downgrade?

Reply:
because of this:
![image](https://github.com/microsoft/FluidFramework/assets/12101885/d0d5e0c1-9e60-41aa-9e22-685284cc29cb)

Reply:
The same downgrade happens in the client group as well.